pkg/kubectl/cmd/delete_test.go

```diff
@@ -375,16 +375,14 @@ func TestDeleteMultipleResourcesWithTheSameName(t *testing.T) {
 }
 
 func TestDeleteDirectory(t *testing.T) {
-	_, svc, rc := testData()
+	_, _, rc := testData()
 
 	f, tf, codec := NewAPIFactory()
 	tf.Printer = &testPrinter{}
 	tf.Client = &fake.RESTClient{
```

Comment:
That's a surreal conversation to follow on email, BTW. :) Yeah, every one of those you pointed out was identical.